pkg/networkutils/network.go

```diff
@@ -551,6 +565,8 @@ func (n *linuxNetwork) buildIptablesConnmarkRules(vpcCIDRs []string, ipt iptable
 	allCIDRs = append(allCIDRs, n.excludeSNATCIDRs...)
 	excludeCIDRs := sets.NewString(n.excludeSNATCIDRs...)
 
+	allCIDRs = utils.DifferenceBetweensStringSlices(allCIDRs, n.includeVPCCIDRsToSNAT)
```

Comment:
I don't understand why you are doing this. I see you get what's in `allCIDRs` that was not in `n.includeVPCCIDRsToSNAT`, but I don't understand why you want this and why you must do it this way.

Maybe the variable should be renamed to reflect its meaning better?